pages/pil/training/schema/certificate.js

```diff
@@ -9,8 +25,8 @@ module.exports = {
     ]
   },
   accreditingBody: {
-    inputType: 'select',
-    options: accreditingBodies,
+    inputType: 'radioGroup',
+    options: accreditingBodyOptions(),
```

Comment:
this is purely me being a pedant, but I really don't like impure functions - if the result of the function needs accreditingBodies, then these should be passed in as an argument, or better still, map them inline here :D